engine-alpha/src/main/java/ea/actor/BodyType.java

```diff
@@ -44,7 +44,7 @@
  * <li>
  * <b>Kinematische</b> Objekte:
  * <ul>
- * <li>Können eine Geschwindigkeit haben, aber onKeyDownInternal nicht auf Kräfte.</li>
+ * <li>Können eine Geschwindigkeit haben, aber reagieren nicht auf Kräfte.</li>
```

Comment:
Das passiert, wenn man in Intellij beim Refactoring nicht aufpasst und vergisst die Suche in Text abzustellen.